### DISCUSSION TITLE
I cannot execute a script using A1 nor click-left

### DISCUSSION THREAD

**Discussion by moshpirit** [polybar/polybar#3120] [Q&A]
I have this rofi wifi menu I want to use that works great when executed from the terminal. I have tried with both methods (using A1, and click-left) and didn't make it work. The module I'm using is wlan, but depends on network-base:

```
[network-base]
type = internal/network
interval = 5
format-connected = <ramp-signal><label-connected>
format-disconnected = 󰤮 <label-disconnected>
inherit = network-base
interface-type = wireless
label-connected = %{A1:wifimenu:}%essid%%{A}
label-disconnected = %{A1:wifimenu}%{F#F0C674}%ifname%%{F#707880} disconnected%{A}

[module/wlan]
inherit = network-base
interface-type = wireless
label-connected = %{A1:wifimenu:}%essid%%{A}
click-left = wifimenu

ramp-signal-0 = %{F#A54242}󰤫 %{F-}
ramp-signal-1 = %{F#A54242}󰤟 %{F-}
ramp-signal-2 = %{F#F0C674}󰤢 %{F-}
ramp-signal-3 = %{F#F0C674}󰤥 %{F-} 
ramp-signal-4 = %{F#F0C674}󰤨 %{F-}
```

What am I missing?

**Comment by patrick96** [comment]
Most likely, the folder containing the `wifimenu` script isn't on the `PATH` when polybar is launched

**Reply by moshpirit** [reply]
Thanks for replying! `wifimenu` is an alias for `~/.scripts/rofi-wifi-menu/rofi-wifi-menu.sh`. Seems like changing it for the long path fixed it. Can we use aliases on polybar or it was just that this specific case wasn't working due to the `~`?

**Reply by patrick96** [reply]
Aliases are very shell-specific and normally only useful on the commandline and can't be used in scripts or other contexts (e.g. from within polybar).
